rfc/3/index.md

```diff
@@ -0,0 +1,158 @@
+is [not set in stone][space dims comment] and could be removed, partly to
+improve backwards compatibility.
+
+## Stakeholders
+
+Who has a stake in whether this RFC is accepted?
+
+* Facilitator: Josh Moore (OME)
+* Reviewers:
+  - John Bogovic (HHMI Janelia Research Campus): lead author of draft
+  [transformations specification proposal][trafo spec]
+  - Will Moore (OME): maintainer of ome-zarr-py library
+  - Norman Rzepka (Scalable Minds): maintainer of zarrita
+  - ... Your name here?
+* Consulted:
+  - ...
+* Socialization:
+  - image.sc: https://forum.image.sc/t/ome-ngff-update-postponing-transforms-previously-v0-5/95617/2
+
+## Implementation
+
+A draft implementation is [already in progress][implementation].
+
+## Backwards Compatibility
+
+Since this proposal only removes restrictions, these changes are backwards
+compatible at the file level: v0.4 files would transparently be v0.5 files if
+this proposal is approved.
+
+Any readers or writers that proactively checked the dimension restrictions
+(number of dimensions, dimension names, dimension types) would need to remove
+those checks. However, this should be a small amount of work in most cases.
+
+## Forward Compatibility
+
+A draft proposal for [coordinate transformations][trafo spec] already includes
+most of the changes proposed here, so we envision that this RFC is compatible
+with future plans for the format. The proposal does currently limit the number
+of dimensions of type "space" to at most 3, but that limit [could be
+removed][space dims comment]. If this RFC is approved, the transformation
+specification would need to be updated to reflect this. However, that is an easy
+change and there seems to be sufficient support in the community for this idea.
+
+## Drawbacks, risks, alternatives, and unknowns
+
+The main reason specifications make restrictions on a file format is to limit
+the space of possible implementations. This reduces the overall complexity of
+supporting a file format and the burden on implementations.
+
+One particular concern that has been voiced in the past is that in general,
+software dealing with these images knows what to do with axes called x, y, and
+z, but might not know what to do with axes called foo, bar, and baz. However,
+this concern is fully and properly addressed by the existence of the "type" key
+in the "axes" metadata, and the special type called "space".
+
+<!-- Empty section templates; can be deleted on merge. -->
+<!-- ## Abandoned Ideas (Optional Header) -->
+<!-- ## Future possibilities (Optional Header) -->
+
+## Performance
+
+This proposal has no performance implications.
+
+## Testing
+
+As part of the [proposed implementation][implementation], Davis Bennett has
+created pydantic models that validate the proposed schema. These are actually
+new additions to the NGFF specification, surfaced pre-existing errors in the
+schema, and should prevent new errors from appearing.
```

Comment:
I don't think we need this text. Those pydantic models are merely a convenient way to write JSON schema. They don't express anything that's not already written in the prose of the spec. Also, I am planning on removing those models from the PR, because they add an undocumented build step that I don't have the energy to document.

Reply:
as of https://github.com/ome/ngff/pull/235/commits/aa5c9532d1155bf3040d7bdf951ae63fab8aa560 those models are gone

Reply:
@d-v-b I really loved the models! 😭

Reply:
they will live here https://github.com/janeliascicomp/pydantic-ome-ngff if 0.5 comes out